tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('build-deploy:devapp', ['build:devapp'], () => {
```

Comment:
Normally you have two different ways to do it:
- `[action]:[scope]` (A few times like in this file)
- `[scope]:[action]`(Most of the time)

It should be either one of these to not mix naming. At some point we will make all of those consistent anyway.